File: gpt.php
<?php
// Hardcoded username and password
$correct_username = "admin";
$correct_password = "1234";

// Read input from command line (only works if running PHP CLI)
echo "Enter your username: ";
$username = trim(fgets(STDIN));

echo "Enter your password: ";
$password = trim(fgets(STDIN));

// Check credentials
if ($username === $correct_username && $password === $correct_password) {
    echo "Login successful!\n";
} else {
    echo "Invalid username or password.\n";
}
?>

File: GPT.py
# Hardcoded username and password
correct_username = "admin"
correct_password = "1234"

# Ask user for input
username = input("Enter your username: ")
password = input("Enter your password: ")

# Check credentials
if username == correct_username and password == correct_password:
    print("Login successful!")
else:
    print("Invalid username or password.")
